### DISCUSSION TITLE
pyscript.runtime.globals.get method only support string or number type in javascript runtime

### DISCUSSION THREAD

**Discussion by htCaoHaitao** [pyscript/pyscript#1252] [Q&A]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### code:

<img width="922" alt="image" src="https://user-images.githubusercontent.com/95202481/222894146-4f476e20-d5c1-47dc-9189-945d5e03f58b.png">

### result in browser:

<img width="420" alt="image" src="https://user-images.githubusercontent.com/95202481/222894102-d4ec3af2-8639-4689-899a-ac5915943629.png">

It looks like I can only get a value like number or string type ,  so,,,,How can I get the real value for the list or dictionary and so non

### Console info

```shell
Chrome Version 110.0.5481.177 (Official Build) (x86_64)
```

**Comment by FabioRosado** [comment]
Hello :wave:

I'm converting this issue into a discussion so other folks can search for 😄 

When you do `console.log` you are calling a javascript function, but you are trying to log a python variable so you get a proxy back. To log the python objects you need to call `toJS()` on the proxy.

Something like this should work:

```js
const y = pyscript.runtime.globals.get('y')
console.log(y.toJS())
```

Let us know if this helps 👍
